content/_releases/22/38.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Product Update - v22.38
+description: Details of the product update v22.38 from 22nd of September 2022.
+layout: rel
+version: "22.38"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-09-22
+redirect_from:
+  - /releases/22.38.html
+---
+
+## Improvements
+
+### Credentials Page: Search by component name
+
+In our quest to improve every aspect of our User Interface, we added a search functionality
+to the Credentials page. When you navigate to Integrate - Credentials you can now
+use the Search by component name to find the credential card you need instead of
+scrolling the page to find it.
+
+{% include img.html max-width="80%" url="/assets/img/RN/2238/RN2238-credentials-search-by-name.gif" title="Credentials Page: Search by component name" %}
+
+### WebHook Step: Send sample request
+
+When you configure WebHook component step in your integration flow, you can either
+**Send sample request** or **Add sample manually**. When you select to send sample
+request a WebHook address is provided which you can use to send your example sample.
+
+We changed the wording to highlight that this address to be used **only one-time** to
+send a representative sample which you can use to configure subsequent steps in your
+integration flow.
+
+{% include img.html max-width="60%" url="/assets/img/RN/2238/RN2238-webhook-send-sample-request.png" title="WebHook step: send sample request" %}
+
+### REST-API-V2 component POST and PUT body type
+
+While configuring the REST-API-v2 component step in integration flows to `POST` or
```

Comment:
This para is confusing - pls delete.

The link is useful, but lets use it in a 2nd para sthg like this:

'This change avoids possible errors when Users forget to change the input field settings.... Ref:Link'